src/uucore/src/lib/features/checksum.rs

```diff
@@ -502,11 +555,13 @@ where
         };
 
         for (i, line) in lines.iter().enumerate() {
-            if let Some(caps) = chosen_regex.captures(line) {
+            let line_bytes = os_str_as_bytes(line).expect("UTF-8 decoding failure");
```

Comment:
The function returns a `UResult`, so you can use `?`:
```suggestion
            let line_bytes = os_str_as_bytes(line)?;
```